manifest.json

```diff
@@ -30,7 +30,7 @@
         "modal.html"
     ],
     "content_scripts": [{
-        "js": ["libs/jquery.min.js", "libs/extLibs.js", "main.js"],
+        "js": ["libs/jquery.min.js", "libs/extLibs.js", "libs/virtual-key-code.min.js", "main.js"],
```

Comment:
@jinhduong It is the predefined constants, not a library. It'll make easier to change the hotkey later. Why don't we add a option page allow users to use the hotkey they want? It would be a great feature.